state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -582,6 +582,10 @@ pub mod pallet {
 	pub(crate) type ScheduledTxForReject<T: Config<I>, I: 'static = ()> =
 		StorageValue<_, Vec<TaintedTransactionDetails<T, I>>, ValueQuery>;
 
+	#[pallet::storage]
+	pub(crate) type BrokerPrivateChannels<T: Config<I>, I: 'static = ()> =
+		StorageMap<_, Twox64Concat, T::AccountId, ChannelId, OptionQuery>;
```

Comment:
This can use the Identity hasher.

```suggestion
		StorageMap<_, Identity, T::AccountId, ChannelId, OptionQuery>;
```